config/locales/en.yml

```diff
@@ -34,6 +34,12 @@ en:
       date_input_fields:
         date_of_birth_label: 'Date of birth'
         date_of_birth_hint: 'For example, 31 3 1980'
+      shared_ownership_form:
+        shared_ownership_item:
+          partner_or_ex_partner:  Yes, a partner or ex-partner
+          housing_assocation_or_landlord: Yes, a housing association or landlord
+          friend_family_member_or_other_individual: Yes, a friend, family member or other individual
+          no_sole_owner: No, I am the sole owner
```

Comment:
This probably needs to be worded differently when a provider uses the screen. Something like 'No, my client is the sole owner'?

Reply:
I have change this to "No, sole owner" to keep it generic